tests/test_include_markdown.py

```diff
@@ -87,6 +87,31 @@
 
 This must be included.
 
+<!-- END INCLUDE -->
+''',
+        ),
+
+        # Start and end arguments with escape sequence
```

Comment:
Can you change this comment by `# Escaped special characters` to be more specific? I think that here it's not necessary to specify "start and end" and "escape secuence" is a bit ambiguous.

Reply:
Check.